examples/components/menu/basic_usage.ts

```diff
@@ -0,0 +1,18 @@
+import {Component} from "angular2/core";
+import {MATERIAL_DIRECTIVES} from "ng2-material/all";
+
+@Component({
+  selector: 'menu-basic-usage',
+  templateUrl: 'examples/components/menu/basic_usage.html',
+  styleUrls: ['examples/components/menu/basic_usage.css'],
+  directives: [MATERIAL_DIRECTIVES]
+})
+export default class MenuBasicUsage {
+ display: boolean = false;
+
+test()
```

Comment:
Yep I did in vscode 

Envoyé de mon iPhone

> Le 26 avr. 2016 à 14:49, Grégory Copin <notifications@github.com> a écrit :
> 
> In examples/components/menu/basic_usage.ts:
> 
> > @@ -0,0 +1,18 @@
> > +import {Component} from "angular2/core";
> > +import {MATERIAL_DIRECTIVES} from "ng2-material/all";
> > +
> > +@Component({
> > +  selector: 'menu-basic-usage',
> > +  templateUrl: 'examples/components/menu/basic_usage.html',
> > +  styleUrls: ['examples/components/menu/basic_usage.css'],
> > +  directives: [MATERIAL_DIRECTIVES]
> > +})
> > +export default class MenuBasicUsage {
> > + display: boolean = false;
> > +
> > +test()
> tslint will warn you but won't auto align stuff by himself.
> 
> there's already a default conf for tslint in the project. Have you configure your text editor to use it ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>